assets/app/views/deployments.html

```diff
@@ -82,68 +82,106 @@
                 {{deployment.metadata.name}}
-                <span ng-if="(deployment | annotation:'deploymentVersion') == deploymentConfig.status.latestVersion">(latest)</span>
+                <span ng-if="deploymentIsLatest(deploymentConfig, deployment)">(active)</span>
+                <span class="text-muted deployment-summary-status">{{describeDeploymentStatus(deployment)}}</span>
+                <span class="deployment-summary-toggler"><a href="" ng-click="detailed = true">details</a></span>
               </h3>
-              <div>
-                <dt>Created:</dt>
-                <dd><relative-timestamp timestamp="deployment.metadata.creationTimestamp"></relative-timestamp></dd>
+            </div>
+            <!-- disabled until we have view logs and metrics 
+            <div class="btn-group">
+              <button type="button" class="btn btn-primary dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                View <span class="caret"></span>
+              </button>
+              <ul class="dropdown-menu">
+                <li><a href="#">Logs</a></li>
+                <li><a href="#">Metrics</a></li>
+              </ul>
+            </div>-->
+          </div><!-- /.row -->
+          <div class="row deployment-details">
+            <div class="col-md-12">
+              <div class="deployment-detail" style="margin-bottom:15px;">
+                <span class="deployment-detail-label">Status:</span>
+                <span ng-switch="deploymentStatus(deployment)" class="hide-ng-leave">
+                  <span ng-switch-when="Complete" class="fa fa-check text-success" aria-hidden="true"></span>
+                  <span ng-switch-when="Failed" class="fa fa-times text-danger" aria-hidden="true"></span>
+                  <span ng-switch-when="New" class="spinner spinner-xs spinner-inline" aria-hidden="true"></span>
+                  <span ng-switch-when="Pending" class="spinner spinner-xs spinner-inline" aria-hidden="true"></span>
+                  <span ng-switch-default class="fa fa-refresh fa-spin" aria-hidden="true"></span>
+                </span>
+                {{deploymentStatus(deployment)}}
+                <span style="margin-left: 7px;">
+                  <button ng-show="!rollBackCollapsed && deploymentStatus(deployment) == 'Complete' && !deploymentIsLatest(deploymentConfig, deployment)" ng-disabled="(deploymentConfigDeploymentsInProgress[deploymentConfigName] | hashSize) > 0" type="button" class="btn btn-primary" ng-click="rollBackCollapsed = !rollBackCollapsed">Roll Back</button>
+                  <div ng-show="rollBackCollapsed" class="well well-sm">
+                    <div class="checkbox">
+                      <label>
+                        <input type="checkbox" ng-model="changeScaleSettings" ng-disabled="(deploymentConfigDeploymentsInProgress[deploymentConfigName] | hashSize) > 0"> include previous deployment's replica count and selector
```

Comment:
When using the rollback CLI command, if no specific version is specified, we choose the latest "complete" deployment whose version is less than the current `deploymentConfig.latestVersion`. If you wanted to get more clear in the console, you could determine the target in the same way and present it explicitly to the user as the default (e.g. "rollback to v4").